src/include/sof/lib/dma.h

```diff
@@ -216,6 +216,11 @@ struct dma_chan_data {
 	void *private;
 };
```

Comment:
@tlauda would appreciate a short explanation of *why* we are doing this change. E.g Move it to header because it is needed by multiple modules, etc.

Reply:
@dbaluta Done. Also updated similar dai commit.